src/components/app/App.js

```diff
@@ -1,24 +1,79 @@
 import React from 'react';
-import logo from '../../logo.svg';
-import './App.css';
 
+import { Header } from '../header/HeaderFromLecture';
+import { Footer } from '../footer/Footer';
+import TestCard, { PostCard as Card } from '../post-card/PostCard';
+// todo: достать в строке 7 массив allComments из констант
+import {allComments} from "../../constants";
+import { postsList, usersList } from '../../constants'; // todo помимо константы postsList достать еще usersList
+// todo: тут сделать импорт  UserCard из components/user-card/UserCard
+import {UserCard} from "../user-card/UserCard";
+import './App.scss';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+// console.log(allComments);
+// console.log(postsList)
+// console.log(usersList);
 function App() {
+  const renderList = () => {
+    const res = [];
+
+    for (let i = 0; i < 6; i++) {
+      const item = postsList[i];
+      res.push(<Card post={item} key={item.id}/>)
+    }
+
+    return res;
+  };
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header />
+      <div>content will appear here soon</div>
+
+      {/*<div className="d-flex posts-container">*/}
+      {/*  {*/}
+      {/*    renderList()*/}
+      {/*  }*/}
+      {/*</div>*/}
+
+      <div className="d-flex posts-container">
+        {
+          postsList.map((post, index) => {
+            const odd = post % 2 !== 0;
```

Comment:
here should be `index % 2` cause post is object and it can be used with `%` operator

Reply:
ok